src/examples/example_database_schema_report.yml

```diff
@@ -0,0 +1,28 @@
+---
+description: >
+  Example of how to run the hmpps/database_schema_report job
+usage:
+  version: 2.1
+  orbs:
+    hmpps: ministryofjustice/hmpps@8
+
+  jobs:
+    initialise_db_schema:
+      executor:
+        name: hmpps/java_localstack_postgres
```

Comment:
any reason for needing localstack here?

Reply:
`./gradlew check` task requires it in our project.

An alternative would be to omit it and force projects (us!) to write a task to initialise the database

Reply:
this is an example of how to get a database schema report though, would have though the example should just need the postgres executor?

Reply:
This example was deleted, as it is fairly simple to run without an example